setup.py

```diff
@@ -41,11 +41,12 @@ def get_packages(package):
 )
 
 env_marker_win = "sys_platform == 'win32'"
-
+env_marker_below_38 = "python_version < '3.8'"
```

Comment:
I find this style a bit odd, but at least it's consistent with what we already have. (I tend to rather see the markers inline, eg `"typing-extensions; python_version < '3.8'"`.)

Reply:
originally this was because we grouped windows / cython markers, I have no preference